mwclient/errors.py

```diff
@@ -97,18 +136,26 @@ def __str__(self):
 
 
 class EmailError(MwClientError):
+    """Base class for email errors."""
     pass
 
 
 class NoSpecifiedEmail(EmailError):
+    """Raised when trying to email a user, that has no email specified."""
     pass
 
 
 class NoWriteApi(MwClientError):
+    """The wiki does not have the write API enabled."""
```

Comment:
this exception is actually never raised any more; all use of it was removed in cd4830a5d10ea452af73a2c098f92222b8145df8 , but we didn't remove the exception itself. maybe remove it in a preparatory commit, rather than documenting it?